istio/discovery.go

```diff
@@ -83,6 +85,16 @@ func (in *Discovery) getControlPlaneConfiguration(kubeCache cache.KubeCache, con
 		return nil, err
 	}
 
+	certConfigMap, err := kubeCache.GetConfigMap(controlPlane.IstiodNamespace, certificatesConfigMapName)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I don't think a failure here should result in the whole function returning an error. Can we log an error or warning if `err != nil` and then parse if there's no error?